common-types/index.d.ts

```diff
@@ -144,5 +149,6 @@ interface DevPortfolioSubmission {
   member: IdolMember;
   openedPRs: string[];
   reviewedPRs: string[];
-  status: 'valid' | 'invalid' | 'pending';
+  openedResults: ValidationResult[];
+  reviewedResults: ValidationResult[];
```

Comment:
Maybe `reviewedPRs` and `openedPRs` can be defined by a list of another custom type instead of adding extra fields to `DevPortfolioSubmission`. Something like:
```
interface PullRequestSubmission {
    status: 'valid' | 'invalid' | 'pending',
    link: string,
    reason?: string
}
```
What are your thoughts on this?

Reply:
It was kinda awkward to separate the results from the PR it corresponds to, so I like this better. It also gives the chance to actually use the `'pending'` status.